opm/simulators/wells/StandardWellV_impl.hpp

```diff
@@ -2029,40 +2029,12 @@ namespace Opm
             }
         }
 
-        // processing the residual of the well control equation
-        const double well_control_residual = res[numWellEq_ - 1];
```

Comment:
The resulting failure is actually because a bug was fixed here by accident.  In this well model, the index of the well control equation is not `numWellEq_ - 1`. It should be `Bhp` or `numStaticWellEq - 1`.